CefSharp.Core/ManagedCefBrowserAdapter.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Stdafx.h"
```

Comment:
Also if we're going to use a `.cpp` file should probably migrate everything over? Little inconsistent otherwise.